config/webhook.yaml

```diff
@@ -87,11 +87,20 @@ spec:
       # high value that we respect whatever value it has configured for the lame duck grace period.
       terminationGracePeriodSeconds: 300
 ---
-apiVersion: v1
-kind: Secret
+apiVersion: sigstore.dev/v1alpha1
+kind: ClusterImagePolicy
 metadata:
-  name: verification-key
-  namespace: cosign-system
-# stringData:
-#   cosign.pub: |
-#     <PEM encoded public key>
\ No newline at end of file
+  name: image-policy
```

Comment:
I didn't realize this until below, but I was expecting the webhook to support N of these CRDs, not a singleton with a fixed name 😅

Reply:
Agreed, 100%, though as a first pass, wanted to start here. I feel like there are questions re: resolution of duplicates (conflicts) across the _n_ policies that needs to be discussed.